modular_skyrat/modules/bonecrusher/bone_crusher.dm

```diff
@@ -0,0 +1,193 @@
+///***BONE CRUSHER***///
+//So basically this is shamelessly copy pasta'd crusher code because I cant figure out a good way to remove a component thats already been
+//initialized by the game, in this case the primary reason would be the adjustment of force_wielded from 20 to 15. I cant exactly figure out
+//how to do this so if someone has a good idea, drop it in the comments and I'll commit it. The only actual changes to this should be on
+//lines 11-16, 36, 38 and lines 48-49. Changes are noted where made.
+//Also yes this is recipe locked to ashwalkers only, since thats the primary demographic this is intended for. Intended to be slightly weaker
+//than the PKC, but better than most weapons ashwalkers can make because of trophies existing. Credit to the original owner of this code.
```

Comment:
This feels like a lot of comment for stuff that probably should've been in the PR body.

Reply:
Temporary comments minus the credit and line edits.